packages/cli/src/fulltext/SourceIndex.ts

```diff
@@ -101,7 +101,7 @@ export class SourceIndex {
       if (verbose()) console.log(`No language found for file: ${fileName}`);
       splitter = new RecursiveCharacterTextSplitter();
     }
-    const filePath = join(directory, fileName);
+    const filePath = isAbsolute(fileName) ? fileName : join(directory, fileName);
```

Comment:
@dustinbyrne I saw this fileName occurring as an absolute path so that when it was joined with directory it was a non-existent path. I'm not sure how this could have happened though, because the file paths come from `git ls-files` and `git status` and those should be relative paths, right?

Reply:
It looks like it may be coming from the old listProjectFiles.